README.md

```diff
@@ -97,34 +97,35 @@ This is the mode that [Babel](https://github.com/babel/babel) is currently using
 
 ### Independent mode (`--independent`)
 
-Independent mode Lerna projects allows maintainers to increment package versions independently of each other. Each time you publish, you will get a prompt for each package that has changed to specify if it's a patch, minor, major or custom change. 
+Independent mode Lerna projects allows maintainers to increment package versions independently of each other. Each time you publish, you will get a prompt for each package that has changed to specify if it's a patch, minor, major or custom change.
 
 Independent mode allows you to more specifically update versions for each package and makes sense for a group of components. Combining this mode with something like [semantic-release](https://github.com/semantic-release/semantic-release) would make it less painful. (There is work on this already at [atlassian/lerna-semantic-release](https://github.com/atlassian/lerna-semantic-release).
 
 > The `version` key in `lerna.json` is ignored in independent mode.
 
 ## Commands
 
-### Init
+### init
```

Comment:
Made more sense to me to just have this be the command name casing - "Ls" for `ls` looks really weird o.O